doc/source/quick_start.md

```diff
@@ -0,0 +1,262 @@
+
+```{code-cell} ipython3
+# Add 1 to the raster array
+rast += 1
+```
+
+Additionally, the class {class}`~geoutils.Raster` possesses a NumPy masked-array interface that allows to apply to it any [NumPy universal function](https://numpy.org/doc/stable/reference/ufuncs.html) and
+most other NumPy array functions, while logically casting {class}`dtypes<numpy.dtype>` and respecting {attr}`~geoutils.Raster.nodata` values.
+
+```{code-cell} ipython3
+# Apply a normalization to the raster
+import numpy as np
+rast = (rast - np.min(rast)) / (np.max(rast) - np.min(rast))
+```
+
+## Casting to {class}`~geoutils.Mask`, indexing and overload
+
+All {class}`~geoutils.Raster` classes also support Python logical comparison operators ({func}`==<operator.eq>`, {func}` != <operator.ne>`, {func}`>=<operator.ge>`, {func}`><operator.gt>`, {func}`<=<operator.le>`,
+{func}`<<operator.lt>`), or more complex NumPy logical functions. Those operations automatically casts them into a {class}`~geoutils.Mask`, a subclass of {class}`~geoutils.Raster`.
+
+```{code-cell} ipython3
+# Get mask of an AOI: infrared index above 0.7, at least 200 m from glaciers
+mask_aoi = np.logical_and(rast > 0.7, rast_proximity_to_vec > 200)
+```
+
+Masks can then be used for indexing a {class}`~geoutils.Raster`, which returns a {class}`~numpy.ma.MaskedArray` of indexed values.
+
+```{code-cell} ipython3
+# Index raster with mask to extract a 1-D array
+values_aoi = rast[mask_aoi]
+```
+
+Masks also have simplified, overloaded {class}`~geoutils.Raster` methods due to their boolean {class}`dtypes<numpy.dtype>`. Using {func}`~geoutils.Raster.polygonize` with a
+{class}`~geoutils.Mask` is straightforward, for instance, to retrieve a {class}`~geoutils.Vector` of the area-of-interest:
+
+```{code-cell} ipython3
+# Polygonize areas where mask is True
+vect_aoi = mask_aoi.polygonize()
+```
+
+```{code-cell} ipython3
+# Plot result
+rast.show(cmap='Reds', cbar_title='Normalized infrared')
+vect_aoi.show(fc='none', ec='k', lw=0.5)
+```
+
+## Saving to file
+
+Finally, for saving a {class}`~geoutils.Raster` or {class}`~geoutils.Vector` to file, simply call the {func}`~geoutils.Raster.save` function.
+
+```{code-cell} ipython3
+# Save our AOI vector
+vect_aoi.save("myaoi.gpkg")
+```
+
+```{code-cell} ipython3
+:tags: [remove-cell]
+import os
+os.remove("myaoi.gpkg")
+```
+
+## Parsing metadata with {class}`~geoutils.SatelliteImage`
+
+In our case, `rast` would be better opened using the {class}`~geoutils.Raster` subclass {class}`~geoutils.SatelliteImage` instead, which tentatively parses
+metadata recognized from the filename or auxiliary files.
+
+```{code-cell} ipython3
+# Name of the image we used
+import os
+print(os.path.basename(filename_rast))
+```
+
+```{code-cell} ipython3
+# Open while parsing metadata
+rast = gu.SatelliteImage(filename_rast, silent=False)
+```
+
+```{admonition} Wrap-up
```

Comment:
Very convincing quick-start section. My only comment would be that maybe we should try to get a more concrete example, that will appeal to the majority of geoscientists. This may require adding other example data. Let's think about it...

Reply:
+1